ShadowDOM/explainer.md

```diff
@@ -313,7 +313,44 @@ This approach is much simpler than [alternate proposals](#alternate-proposals) a
 This approach does have a few limitations:
 - The `<style>` definition *must* occur before it is imported, otherwise the import map will not be populated. Based on developer feedback, this is not a major limitation.
 - Since Import Maps have no knowledge of an underlying type for their mappings, declarative modules with the same specifier (e.g. "foo"), but differing types (e.g. one JavaScript module with a specifier of "foo" and one CSS module with a specifier of "foo") would create separate entries in the generated import map, and only the first definition would actually be mapped. See [Open Issues](#open-issues) for some potential solutions to this scenario.
+- The data URI must be URL-encoded, because many CSS selectors have special meaning in URLs. One example is the `#` ID selector in CSS, which is a fragment identifier in URLs and can only
+exist once in a URL.
 
+Alternatively, a Blob URL could be used instead of a data URI. Using a Blob URL offers several performance advantages over a data URI, such as avoiding URL-encoding and a much
+smaller Import Map value string stored in memory. 
```

Comment:
I don't see an argument against Blob URLs presented here. If we only have positive things to say about this option, why not change the proposal to only suggest Blob URLs, and move Data URIs to the rejected alternatives section?